packages/@webex/webex-core/src/lib/services/services.js

```diff
@@ -166,6 +168,15 @@ const Services = WebexPlugin.extend({
     this._serviceUrls = {...this._serviceUrls, ...serviceUrls};
   },
 
+  /**
+   * saves the host catalog
+   * @param {Object} hostCatalog
+   * @returns {void}
+   */
+  _updateHostCatalog(hostCatalog) {
```

Comment:
do we not have any changes to tests?

Reply:
we do...I missed adding it here in the next branch. Added now